modules/YetiSimulator/board_support/evse_board_supportImpl.cpp

```diff
@@ -0,0 +1,121 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Pionix GmbH and Contributors to EVerest
+
+#include "evse_board_supportImpl.hpp"
+
+namespace module {
+namespace board_support {
+
+void evse_board_supportImpl::init() {
+}
+
+void evse_board_supportImpl::ready() {
+    const auto capabilities = types::evse_board_support::HardwareCapabilities{
+        .max_current_A_import = 32.0,
+        .min_current_A_import = 6.0,
+        .max_phase_count_import = 3,
+        .min_phase_count_import = 1,
+        .max_current_A_export = 16.0,
+        .min_current_A_export = 0.0,
+        .max_phase_count_export = 3,
+        .min_phase_count_export = 1,
+        .supports_changing_phases_during_charging = true,
+        .connector_type = types::evse_board_support::Connector_type::IEC62196Type2Cable};
```

Comment:
Designated initializers are only supported in C++20 and above.

(Also below with `return {.ampacity = Ampacity::None};`.)